Automattic/wp-calypso
# Include an Audit Script for the Colors Used in SVG Images

**…and Update the Remaining Images**

This pull request introduces an auxiliary script that detects the use of non-standard color values in the SVG images found in the repository.

This script has been tested with the updates performed in #36510 (as a part of #36097). Some images were left out from that PR due to quirky color substitution, so they have been updated in this PR instead (along with the tweaks to the script). Our retired plan icons have been removed instead of receiving the facelift.

### Testing instructions

* Make sure running following command doesn’t throws errors:

    ```sh
    node ./bin/audit-svg-colors.js
    ```

* Ensure the updated images show up in Calypso.


---

☝️ @sixhours @davewhitley @Aurorum
